.github/workflows/cd-pipeline.yaml

```diff
@@ -138,24 +152,55 @@ jobs:
           # HACK: Obtain chart tarball out of the latest commit
           chart_version="$(yq ".version" charts/bitnami/${chart}/Chart.yaml)"
```

Comment:
I think this can be fixed now, by using `jq`:
```
chart_version=$(jq -re '.actions|map(select(.action_id == "helm-publish"))[0] | .application.version' result.json)
```
Where `result.json` is part of the artifacts from the publish job.

Then, we will also be able to remove the checkout step for the master branch (line 126)

Reply:
I have used a new logic and removed the `checkout` step as well as the dependency with the `get-chart` step 🚀 
Thanks for the feedback